README.md

```diff
@@ -1,103 +1,44 @@
 
-    ```sh
-    aws s3api create-bucket --bucket example-bucket-tf-state --region ap-southeast-2 --create-bucket-configuration LocationConstraint=ap-southeast-2 --acl private
-    ```
+## Creating a new environment
 
-8. Go into the `terraform/` directory. From there, run `terraform init`. It will ask you for some S3 config. Specify the
-location where your remote state lives. For demo.rr-staging.click, this will be:
+If you want to spin up a brand new deployment of Rabble Rouser, [go here](./docs/new_environment.md).
 
-    ```
-    bucket: rr-staging.click-tf-state
-    key: terraform.tfstate
-    region: ap-southeast-2
-    ```
+## About the `tf` script
 
-9. **Note: This is only necessary for a brand new environment.** If you want email sending to work, you need to set up SES. See the instructions in [mailer](https://github.com/rabblerouser/mailer) for how to do that.
-
-## What did that just do?
-
-This whole process will create a local file `.terraform/terraform.tfstate`, to remember where your state is stored. You can ignore
-this file, and if you lose it, you can regenerate it with the above command.
-
-Notes on the S3 bucket:
- - The bucket name must be unique to the whole world (not just your AWS account)
- - Terraform will store all of its state in the bucket, some of which is sensitive, so don't share access with anyone who doesn't need it
- - Terraform won't manage the bucket itself, just its contents
-
-## Create all your infrastructure
-
-1. Edit the file `terraform.tfvars` so that it references the Rabble Rouser instance you want to manage. The values that
-are in there by default are specific for the project's staging instance at https://demo.rr-staging.click. You can also
-override any of the variables in `variables.tf` if you don't like their default values.
-
-2. Do a dry run first to see what terraform will do:
-
-    ```
-    terraform plan
-    ```
-
-3. Create the infrastructure: *Warning: this will cost you money, even if you have the AWS free tier!*
-
-    ```
-    terraform apply
-    ```
-
-## Re-run the provisioner
-
-You might want to do this because the ansible code has changed, or to deploy the latest version of the app:
-
-```sh
-./re-deploy.sh
-```
-
-## To seed the database
-
-Before the application can be useful, the database needs some seed data. The seed script can be run as part of the
-provisioner, but because it is not idempotent (i.e. you should only run it once), you must opt-in to it:
+Throughout these docs, you are often told to run a special `tf` script, rather than running terraform directly. This is
+to work around some terraform limitations ([1](https://github.com/hashicorp/terraform/issues/10462),
+[2](https://github.com/hashicorp/terraform/issues/5190)), which currently make the native terraform commands a bit
+verbose. You can run the script with no arguments to see how to use it, or read [`the source code`](./terraform/tf) to
+see how it works:
```

Comment:
I reckon it would be good to have a note that you need to run `terraform get` even for it to find the local module files. This isn't clear in the output that terraform gives you, it implies that `terraform get` is only for downloading modules.